### ISSUE TITLE
I think it is better to return a collect when random takes a value, laravel is not a direct return value.

### ISSUE THREAD

**Issue by xiaobing1024** [ecrmnn/collect.js#202]
I think it is better to return a collect when random takes a value, laravel is not a direct return value. Uniform data format returned in this way

**Comment by ecrmnn** [comment]
Hi @xiaobing1024 

I'm not sure I follow. `random` in Collect.js works exactly the same way as in Laravel Collection.

**Retrieving a single radom value**
```php
// Laravel (PHP)
collect([1, 2, 3])->random();
//=> 2
```
```js
// Collect.js (JS)
collect([1, 2, 3]).random();
//=> 2
```

**Retrieving mutiple random values**
```php
// Laravel (PHP)
$randoms = collect([1, 2, 3, 4, 5, 6])->random(3);
// => Returns a new collection containing 3 random values

$randoms->all();
//=> [1, 2, 5]
```
```js
// Collect.js (JS)
const randoms = collect([1, 2, 3, 4, 5, 6]).random(3);
// => Returns a new collection containing 3 random values

randoms.all();
//=> [1, 2, 5]
```

**Comment by xiaobing1024** [comment]
Sorry, I got it wrong.

```
// Laravel (PHP)
collect([1, 2, 3])->random();
//=> 2
```

```
// Laravel (PHP) this is different
collect([1, 2, 3])->random(1);
//=> collect(2)
```

```
// Collect.js (JS)
collect([1, 2, 3]).random();
//=> 2
```

```
// Collect.js (JS)
collect([1, 2, 3]).random(1);
//=> 2
```

**Comment by ecrmnn** [comment]
I see! That shouldn't happen.
Fixed in b5b64f5.
Thanks for creating this issue.
